### ISSUE TITLE
Add status to pods/nodes

### ISSUE THREAD

**Issue by moshloop** [flanksource/canary-checker#1120]
Add a status property and a reason

**Comment by yashmehrotra** [comment]
@moshloop The status property is pretty vast, and JSON.Stringifying the entire object would look ugly

I was thinking we can do all the conditions as `<cond-name>:  "status (True or False with error)"` and `phase`. Any other fields that should be there ?

```json
{
  "conditions": [
    {
      "lastProbeTime": null,
      "lastTransitionTime": "2023-07-04T09:54:48Z",
      "status": "True",
      "type": "Initialized"
    },
    {
      "lastProbeTime": null,
      "lastTransitionTime": "2023-07-04T09:55:16Z",
      "status": "True",
      "type": "Ready"
    },
    {
      "lastProbeTime": null,
      "lastTransitionTime": "2023-07-04T09:55:16Z",
      "status": "True",
      "type": "ContainersReady"
    },
    {
      "lastProbeTime": null,
      "lastTransitionTime": "2023-07-04T09:54:48Z",
      "status": "True",
      "type": "PodScheduled"
    }
  ],
  "containerStatuses": [
    {
      "containerID": "containerd://97eabb8b8bc8b9bc49ce9f7ecee5c398681335074b089508226829dacae99a78",
      "image": "docker.io/flanksource/config-db:v0.0.128",
      "imageID": "docker.io/flanksource/config-db@sha256:63c7ab5dedaa40294a4e73559c94f59a7be6046f6dd8cfe264989fe94f471985",
      "lastState": {},
      "name": "config-db",
      "ready": true,
      "restartCount": 0,
      "started": true,
      "state": {
        "running": {
          "startedAt": "2023-07-04T09:55:16Z"
        }
      }
    }
  ],
  "hostIP": "10.0.4.23",
  "phase": "Running",
  "podIP": "10.0.4.178",
  "podIPs": [
    {
      "ip": "10.0.4.178"
    }
  ],
  "qosClass": "Burstable",
  "startTime": "2023-07-04T09:54:48Z"
}

```
